markdown/usage/data_guidelines.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Data Management Guidelines
+subtitle:
+---
+
+## Data management
+
+### Data Management Plan
+
+A Data Management Plan (DMP) is a revisable document explaining how you intend to handle new and existing data, during and following the conclusion of your research project. It is wise to write a DMP as early as possible, using either a tool provided by your host institution or [DS Wizard](http://dsw.scilifelab.se/) ([users outside Sweden](https://ds-wizard.org/)). Ethical and legal considerations regarding the data will depend on where the research is conducted, this is especially true for projects including sensitive human data. For more  information about the Swedish context, please review this page on [Sensitive personal data](https://scilifelab-data-guidelines.readthedocs.io/en/latest/docs/general/sensitive_data.html).
+
+### Data storage and computational resources
+
+To estimate computational resources needed for a specific pipeline see docs/output page ...
+
+/ Feedback from nf-core team needed /: Insert statement/instructions on where or how to get the estimated file sizes, folder structures etc.
```

Comment:
```suggestion
To estimate computational resources needed for a specific pipeline please see the pipeline documentation. This lists the different output filetypes you can expect.
In the future we hope to automate a full run of each pipeline after every release. The pipeline docs will then show a full set of results from a real run, along with all file sizes. This can then be used as a guide as to what to expect for your data.
```